src/utils.ts

```diff
@@ -191,6 +198,32 @@ export class Utils {
         }
     }
 
+    /**
+     * If repository input was set, extract CLI download details from the first JF_ENV_* environment variable.
+     * @param repository - Remote repository in Artifactory pointing to https://releases.jfrog.io/artifactory/jfrog-cli/
```

Comment:
The description of the ```repository``` may be a bit confusing, because it mentions ```https://releases.jfrog.io/artifactory/jfrog-cli/``` - which is actually the default, of no ``` repository``` param was sent. I suggest we improve the description and make it clearer.

Reply:
https://releases.jfrog.io/artifactory/jfrog-cli/ is not only the default, it is the only option